force-app/main/default/core-functionality/internal-components/lwc/crmMessagingQuickText/crmMessagingQuickText.js

```diff
@@ -18,6 +21,7 @@ export default class crmQuickText extends LightningElement {
 
     @api comments;
     @api required = false;
+    @api resetTextTemplate;
```

Comment:
Denne burde ha en default value, sikkert empty
```suggestion
    @api resetTextTemplate = '';
```